schemas/context/profilestitchidentity.example.1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "@id": "https://data.adobe.io/entities/profileStitchIdentity/1",
+  "xdm:namespace": {
+    "xdm:code": "AAM"
```

Comment:
Don't you also need an `xdm:xid` here?  What use is the namespace without the thing you are namespacing?

Reply:
Wouldn't this as-is namespace `@id`.

Reply:
@id is the ID of that particular instance of that schema - if that is what you are doing, then yes, I would agree.  I thought you were referring to some other object...which would be via xdm:id